src/platform/linux/input.cpp

```diff
@@ -1903,6 +1963,8 @@ namespace platf {
    */
   platform_caps::caps_t
   get_capabilities() {
-    return 0;
+    platform_caps::caps_t caps = 0;
+    caps |= platform_caps::pen_touch;
```

Comment:
You also have to implement `pen()` or Moonlight will start sending native pen events that just get ignored.

Reply:
Will implement but will require external testing since I don't have anything capable of doing pen events.

Reply:
I can test with my devices